src/redux/search.js

```diff
@@ -69,9 +70,8 @@ export function searchEpic(action$, { getState }) {
     .switchMap((action) => {
       const { okapi } = getState();
       const { searchType, query, options } = action;
-      const { endpoint, defaults, recordsKey = searchType } = options;
+      const { endpoint, recordsKey = searchType } = options;
 
-      const searchQuery = { ...query, ...defaults };
       const url = `${okapi.url}/${endpoint}?${queryString.stringify(query)}`;
```

Comment:
`defaults` is totally meant to be used (since RM-API requires certain params). This is actually not working in production now because of my mistake here.

This line should instead be: `queryString.stringify(searchQuery)`

Reply:
Although it should be:
```js
const searchQuery = { ...defaults, ...query }
```
So the query isn't overridden by the defaults

Reply:
Fixed.